core/org.osate.xtext.aadl2/src/org/osate/xtext/aadl2/formatting2/Aadl2Formatter.xtend

```diff
@@ -2226,9 +2228,9 @@ class Aadl2Formatter extends PropertiesFormatter {
 			val annexObject = parseAnnexObject.apply(annexParser, trimmedText)
 			val annexParseResult = AnnexParseUtil.getParseResult(annexObject)
 			if (annexParseResult !== null) {
-				//Get the injector for the annex.
-				val grammarName = annexParseResult.rootNode.grammarElement.getContainerOfType(Grammar).name
-				val annexInjector = OsateCorePlugin.^default.getInjector(grammarName)
+				// Get the injector for the annex.
+				val grammar = annexParseResult.rootNode.grammarElement.getContainerOfType(Grammar)
+				var annexInjector = getInjector(grammar)
```

Comment:
Why was `annexInjector` switched from `val` to `var`?

Reply:
typo